spec/services/admin/participants/search_spec.rb

```diff
@@ -124,6 +133,48 @@
           expect(results).not_to include(pp_2, pp_3)
         end
       end
+
```

Comment:
@peteryates Here is where i was having issues with the school tests.

Reply:
Yeah, this is going to get ugly because like in the diagram above we're already joining `participant_profiles` to `schools`.

I think the cleanest approach is to remove NPQ applicants and move them under the NPQ tab.